starlight/static/sass/App.scss

```diff
@@ -0,0 +1,20 @@
+@charset 'UTF-8';
+
+// 1. Variables
+@import "variables";
+
+// 1. Vendors
+$icon-font-path: '~bootstrap-sass/assets/fonts/bootstrap/';
```

Comment:
No need to add `bootstrap-sass` or `font-awesome` things here. Bootstrap is not required for this project (Semantic UI is), and Font Awesome already comes by default with Semantic UI.

Reply:
We will however need to set up imports to the proper Semantic UI SASS files here, so that Webpack will pack it up and include it for us. Come see me if you need pointers for this.